packages/tc-markdown-parser/lib/parse-multiline-definition.js

```diff
@@ -0,0 +1,176 @@
+class SyntaxError extends Error {}
+
+/*
+ This function divides from mutipile definition to code and property string e.g,
+
+ The definition:
+ - child-code
+   propNameOne: propValueOne
+
+ Divides to:
+ {
+   code: 'child-code',
+   propDef: '  propNameOne: propValueOne\n',
+ }
+*/
+const dividePropertyDefinition = subdocument => {
```

Comment:
Are there tools built into mdast that can do this without us having to manually slice it up @chee ?

Reply:
thanks for inviting me to this pull request, this is all such nice code. and so wonderfully commented!!

i don't think there is something like this, i don't think `remark` even has a plugin for kramdown-style definition lists.